### PR TITLE
Updated Micro for variable dimension input

### PR DESCRIPTION
**Pull request by j-varun**

### CODE REVIEW

**Comment by ahundt** [comment]
@j-varun you'll need to resolve the conflicts here

File: enas/cifar10/micro_child.py [hunk 1]
```diff
@@ -245,12 +245,13 @@ def _model(self, images, is_training, reuse=False):
 
     with tf.variable_scope(self.name, reuse=reuse):
       # the first two inputs
+      inp_c = self._get_C(images)
       with tf.variable_scope("stem_conv"):
-        w = create_weight("w", [3, 3, 3, self.out_filters * 3])
+        w = create_weight("w", [inp_c, inp_c, inp_c, self.out_filters * 3])
```
Comment (ahundt) [inline enas/cifar10/micro_child.py]: inp_c might benefit from some other variable name, I have no idea what it means. input channels? maybe just call it `input_channels`?

**Review by ahundt — CHANGES_REQUESTED** [review]
variable naming issue, can you also create a .sh file for running micro search on fashion_mnist like I did for macro search?

**Comment by j-varun** [comment]
Resolved the conflict and made the changes as requested

**Review by ahundt — APPROVED** [review]
LGTM

**Pull request merged**